models/wrf/tutorial/README.rst

```diff
@@ -457,7 +459,7 @@ changes:
 +====================+======================================+=========================================================================================================================================================================================================================================================+
 | driver.csh         | datefnl = 2017042712                 | Change to the final target date; here the final date is already set correctly for this tutorial.                                                                                                                                                        |
 +--------------------+--------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| gen_retro_icbc.csh | datefnl = 2017043000                 | This is the final date to create WRF initial/boundary conditions for. This is set to the last date that files are included in the tutorial.                                                                                                             |
+| gen_retro_icbc.csh | datefnl = 2017042712                 | Set to the final target date of the tutorial.  However, it is possible (not necessary) to create WRF initial/boundary conditions to 2017043000. This is the latest date that files are included in the tutorial.                                        |
```

Comment:
No need to run `gen_retro_icbc.csh` all the way to 2017043000 when tutorial only runs to 2017042712.  This helps streamline troubleshooting.

Reply:
Nice catch here